bucket/lokalise.json

```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://github.com/lokalise/lokalise-cli-2-go",
+    "description": "Lokalise CLI v2 allows you to manipulate any object or data in your Lokalise workspace, which includes files, projects, keys, translations, comments, contributors, teams and more.",
+    "version": "2.5.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/lokalise/lokalise-cli-2-go/releases/download/v2.5.1/lokalise2_windows_x86_64.zip",
+            "hash": "e9cab6a260ba5ffa7fe9e4827ab10048accdb75fbec688a773948b1656a02ff4"
+        }
+    },
+    "bin": "lokalise2.exe",
```

Comment:
```suggestion
    "bin": [
        "lokalise2.exe",
        [
            "lokalise2.exe",
            "lokalise"
        ]
    ],
```

Reply:
I'm sorry but we cannot add alias as `lokalise`. It will collide with [the old lokalise client (v1)](https://docs.lokalise.com/en/articles/1400697-lokalise-cli-v1) or with the npm module binary.

Reply:
So you will install `lokalise` manifest but you cannot use `lokalise` executable?

That does not make sense.

Reply:
Ok, got it. I renamed the manifest `bucket/lokalise.json -> bucket/lokalise2.json`.

Reply:
FYI @Ash258